src/github.com/getlantern/connpool/connpool.go

```diff
@@ -9,9 +9,7 @@ import (
 )
 
 const (
-	DefaultClaimTimeout         = 10 * time.Minute
-	DefaultRedialDelayIncrement = 50 * time.Millisecond
-	DefaultMaxRedialDelay       = 1 * time.Second
+	DefaultClaimTimeout = 10 * time.Minute
```

Comment:
I don't have a strong feeling either way.  The reason that I typically export constants is that it can be a useful form of documentation.  And the reason I made these constants rather than inline was to describe what these numbers mean.